src/Components/Styles/Center.js

```diff
@@ -0,0 +1,6 @@
+import styled from "styled-components";
```

Comment:
I'm not sure if you really need a component for just one style. If thats your way, keep doing it. For me it would be a bit confusing...

Reply:
i will use it later again :)